isis/src/chandrayaan1/objs/Chandrayaan1M3Camera/unitTest.cpp

```diff
@@ -105,26 +105,6 @@ int main(void) {
     cout << "RightAscension = " << cam->RightAscension() << endl;
     cout << "Declination = " << cam->Declination() << endl;     
     
-    // Test a Level-2 image
-    cout << endl << "Testing a Level-2 CH1 cube: " << endl << endl; 
-
-    Cube c2("$chandrayaan1/testData/CH1_xBandLvl2.cub", "r");
-    Camera *cam2 = CameraFactory::Create(c2);
-
-    // Just test the center pixel to make sure the Camera still works on Level-2 
-    // images
-    
-    cout << "For a central pixel position ..." << endl;
-    samp = 2000; 
-    line = 2500; 
-
-    if (!cam2->SetImage(samp, line)) {
-      cout << "ERROR" << endl;
-      return 0;
-    }
-    else {
-      cout << "SetImage succeeded." << endl; 
-    }
```

Comment:
I'll let @scsides answer this because he's the one that uncovered the issue, but I believe that it was removed because it was testing the Chandrayaan camera model on an LRO MiniRF image and that didn't really make sense.

Reply:
Because the test was hitting a Chandrayaan MiniRF cube inside an M3 test, the LRO MiniRF tests also test Chandrayaan1 cubes.